README.md

```diff
@@ -270,7 +271,7 @@ directory.
   }
   ```
 
-  In the example you can also see another convenience property `actions` with all the html-reporter **Redux** actions.
+  In the example you can also see another convenient properties `actions` and `pluginConfig` with all the html-reporter **Redux** actions and plugin configuration.
```

Comment:
Maybe you should not try to write a challenging sentence? You can simply break it on points:
``` md
text-1:
- text-2
- text-3
```